Cargo.toml

```diff
@@ -75,9 +75,12 @@ priority-queue = "1.3.0"
 http-body-util = "0.1.0-rc.2"
 http-body-04 = { package = "http-body", version = "0.4" }
 http-body-1 = { package = "http-body", version = "1.0.0-rc.2" }
-tower-hyper-http-body-compat = { version = "0.2" , features = ["server", "http2"] }
+tower-hyper-http-body-compat = { version = "0.2", features = ["server", "http2"] }
 futures-util = "0.3.26"
 chrono = "0.4.23"
+nix = "0.26.2"
+mockall_double = "0.3.0"
```

Comment:
fair, i'll see what i can do here

Reply:
Using a concrete impl instead of a mock will also break IDEs/test runs outside of Linux FWIW - I don't much care but people have complained about that in the past.